### ISSUE TITLE
Relative inter-document(html to html) links in interactive plantuml diagrams do not work when using data-uri

### ISSUE THREAD

**Issue by sachman** [asciidoctor/asciidoctor-diagram#356]
Hi,
Interdocument navigation works perfectly when I refer to another local document in plantuml diagram as follows:

```
[plantuml%interactive, {autogendir}highLevelBrkDwn, svg, subs=attributes+]
....
@startwbs
* Software Development
** [[{autogen_to_basedir}/public/StandardProductImprovement.html{Go to Process Document} Standard Product Improvement]]
*** [[https://google.com Feature Addition/Improvement]]
*** [[ Bug Fix]]
** [[ New Product Development]]
** [[ Internal(Organizational) Requirements]]
@endwbs
....
```

![image](https://user-images.githubusercontent.com/17062337/120912157-8f8c2b00-c6aa-11eb-8334-5a37709e1c99.png)

However, when I use `:data-uri:`, clicking the local document link leads to a blank blocked page in the browser, as follows:
![plantuml_doclink_datauri](https://user-images.githubusercontent.com/17062337/120912171-b8acbb80-c6aa-11eb-9e6a-05246be28bd2.JPG)

What am I doing wrong? 

UPDATE:

`[[{docdir}/../public/StandardProductImprovement.html Standard Product Improvement]]`

Doing this resolves the link now. However, the link only opens by right clicking and opening in new tab. 

Also, as the path is now getting absolute, it will not be resolved outside the system in which its generated.

**Comment by pepijnve** [comment]
I don't think this has anything to do with this extension directly. The `:data-uri:` attribute does not impact the SVG that's generated by PlantUML. It's the HTML5 backend that handles the encoding of the SVG in a `data:` URI.

What you might be bumping into is how the browser resolves relative URIs located inside an SVG document that has been embedded as a `data:` URI. From what I can tell this simply doesn't work. Safari for instance shows me this error for the relative link. The absolute one to google works fine.
<img width="431" alt="image" src="https://user-images.githubusercontent.com/225326/121506850-69d18f80-c9e4-11eb-9254-576334600213.png">

**Comment by pepijnve** [comment]
Using `inline` instead of `interactive` does seem to work. Is that an option for you? This embeds the SVG content directly in the generated HTML instead of using an `<object>` tag with a `data:` uri. That way the base URI of the SVG document is well defined and relative URI resolution seems to work correctly.

**Comment by sachman** [comment]
Hi @pepijnve ,
Thanks for helping me out.

`inline` works great for me and it solved a major missing piece, though I am not aware of any other limitations that it may pose in future.

I was not sure where to get help in this context, hence asciidoctor-diagram was what I could see. I am sorry if I have spammed.

I have edited the issue title to what I could understand from your statements. 

As the issue is not relevant to this plugin, I am closing this. 

Thanks again for your help.
